assignments/array-methods.js

```diff
@@ -56,28 +56,51 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(runner => fullName.push (runner.first_name + " " + runner.last_name))
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+runners.map(upName => allCaps.push (upName.first_name.toUpperCase()))
```

Comment:
map creates a new array in place - no need to push to another one. Make sure to delete the empty allCaps array on line 64 if you're going to name your map results allCaps.
```suggestion
const allCaps = runners.map(runner => {
  return {
  ...runner,
  first_name: runner.first_name.toUpperCase()
  }
})
```